orttraining/orttraining/test/python/orttraining_distributed_tests.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sys
+import argparse
+
+from _test_commons import run_subprocess
+
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s %(name)s [%(levelname)s] - %(message)s",
+    level=logging.DEBUG)
+log = logging.getLogger("Build")
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--cwd", help="cwd")
+    return parser.parse_args()
+
+def main():
+    import torch
+    ngpus = torch.cuda.device_count()
+
+    if ngpus < 2:
+        raise RuntimeError("Cannot run distributed tests with less than 2 gpu.")
+
+    args = parse_arguments()
+    cwd = args.cwd
+
+    log.info("Running distributed tests pipeline")
+
+    return 0
```

Comment:
this doesn't seem to run any tests. a TODO comment might be useful

Reply:
This pull requests is just for the pipeline. I have distributed tests in another pull request, which I am going to call from here.

I will add a TODO so that it is clear.